app/models/pxe_server.rb

```diff
@@ -34,7 +34,7 @@ def discovered_images
 
   def default_pxe_image_for_windows=(image)
     image.update(:default_for_windows => true)
-    clear_association_cache
+    self.instance_variable_set :@association_cache, {}
```

Comment:
Added in:

```
commit a06e1990d9277922486802b80caf728c7489a678
Author: Jason Frey <jfrey@redhat.com>
Date:   Thu Sep 6 20:48:10 2012 +0000

    Changed modeling so that the WinPE PxeImage is not needed to be selected during provisioning

    - Connected WindowsImage to a PxeImageType in parallel to how PxeImages are connected
    - Added default_for_windows? column to PxeImage to allow user to specify if the image is for windows booting
    - Added support to provisioning to allow a windows image or a pxe image to be put in the pxe_image_id field.  If a WindowsImage is selected, we use the default_for_windows image as the PxeImage.

    BugzID: 15966

    git-svn-id: http://miq-ubuntusub.manageiq.com/svn/svnrepos/Manageiq/trunk@38738 3c68ef56-dcc3-11dc-9475-a42b84ecc76f
```

Reply:
Prefer `.reload` here